slides/02-data-budget.qmd

```diff
@@ -167,20 +176,20 @@ We will talk about more powerful ways of splitting, like stratification, later
 ## Accessing the data `r hexes("rsample")`
 
 ```{r}
-frog_train <- training(frog_split)
-frog_test <- testing(frog_split)
+taxi_train <- training(taxi_split)
+taxi_test <- testing(taxi_split)
 ```
 
 ## The training set`r hexes("rsample")`
 
 ```{r}
-frog_train
+taxi_train
 ```
 
 ## The test set `r hexes("rsample")`
 
 ```{r}
-frog_test
+taxi_test
```

Comment:
I find it weird we are like "don't look at the testing data", and then we go around and looks at it 😆 

Maybe we should add a "don't try this at home" for this slide? 😄

Reply:
I've seen this slide as "the thing you get back from `testing()` is a data frame", not "look what you can find in the testing data". There are big disclaimers when using the test set to evaluate performance.